docs/source/ingest/destination_connectors/code/bash/elasticsearch.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+EMBEDDING_PROVIDER=${EMBEDDING_PROVIDER:-"langchain-huggingface"}
+
+unstructured-ingest \
+	local \
+	--input-path example-docs/book-war-and-peace-1225p.txt \
+	--output-dir local-output-to-elasticsearch \
+	--strategy fast \
+	--chunk-elements \
+	--embedding-provider "$EMBEDDING_PROVIDER" \
+	--num-processes 2 \
+	--verbose \
+	elasticsearch \
+	--hosts "$ELASTICSEARCH_HOSTS" \
+	--username "$ELASTICSEARCH_USERNAME" \
+	--password "$ELASTICSEARCH_PASSWORD" \
+	--index-name "$ELASTICSEARCH_INDEX_NAME"
```

Comment:
did you want to add --num-processes for destination since it's a little unintuitive

Reply:
[remove kwargs, inherit fields, add num-processes for upload to docs](https://github.com/Unstructured-IO/unstructured/pull/2152/commits/04fe747b437792ae68705a6ceed9970f9d6c221b)